docs/Build.md

```diff
@@ -0,0 +1,72 @@
+Naggum Build Guide
+==================
+
+Windows
+-------
+
+To build Naggum under Windows, just use Visual Studio or MSBuild like that:
+
+    $ cd naggum
+    $ nuget restore
+    $ msbuild /p:Configuration=Release Naggum.sln
+
+Linux
+-----
+
+See general build instructions for Linux in the file `.travis.yml` inside of the
+Naggum root source directory.
+
+You'll need [Mono][mono], [NuGet][nuget] and [F# Compiler][fsharp] installed in
+you current environment. Some of them may or may not be part of your Mono
+installation; just make sure you have them all properly installed.
+
+Below is an example of setting up these tools on [NixOS Linux][nixos]; feel free
+to add instructions for any other non-standard distributives.
+
+### NixOS Linux
+
+First of all, install `mono` and `fsharp` packages to your environment:
+
+    $ nix-env -i mono fsharp
+
+Please note that you need `fsharp` >= 4.0 to build Naggum, which is currently on
```

Comment:
Judging from the NixOS manual, you should say *in* the channel, not *on* it. Also, I'd say "unstable channel" is enough—it's obvious that we're talking about NixOS; and make it *the* channel, 'cause as far as I can tell, there's only one.